src/Input/Select/Select.test.tsx

```diff
@@ -360,3 +360,196 @@ it('up and down arrow keys can be used to navigate the options menu', () => {
   fireEvent.keyDown(selectInput, arrowUpEvent);
   expect(firstOption).toHaveClass('active');
 });
+
+describe('<Select/> on click outside', () => {
+  it('should lose focus', async () => {
+    const Component = () => (
+      <React.Fragment>
+        <Select label="ClickOutside">
+          <Select.Option value="ClickOutside">ClickOutside</Select.Option>
+        </Select>
+        <div>outside</div>
+      </React.Fragment>
+    );
+
+    const { queryByTestId, queryByRole, queryByText } = render(<Component />);
+    const selectInput = queryByRole('combobox') as HTMLInputElement;
+    const selectList = queryByTestId('listbox');
+    const outside = queryByText('outside');
+
+    fireEvent.focus(selectInput);
+    expect(selectList.hasAttribute('open')).toEqual(true);
```

Comment:
should we add `expect(selectInput).toHaveFocus();` to correspond to line 384?

Reply:
will remove `toHaveFocus`, fireEvent.focus doesn't update jsdom.